sag_et.par

```diff
@@ -4,195 +4,241 @@
 ! s = list of subdivisions for above division using non-native fifth
 !
 n 5 3
+/||\
 n 7 4
+/||\
+n 9 5
+|\
 n 10 6
-/|)  /N\
+(|\  /||\
 d 12 7
 s 2 3 4 6
-/N\
+/||\
 n 14 8
 |\
 n 15 9
-/|  N\  /N\
+/|  ||\  /||\
 n 16 9
-)N(  M)
+|)  (|\

```

Comment:
I noticed that, for most of the new limma fraction based notations, more symbols were provided than are necessary to notate these EDOs. You can see how I am figuring this here: https://docs.google.com/spreadsheets/d/1Xnm2yNQ2C8QuD7Di-Ik7ttmni-AjFmAkDN2clQbzuuo/edit?usp=sharing

For example, for 16EDO, there are two steps to the limma and one step to the apotome. Following the implicit logic (as best as I could discern it) of the very small sets of symbols provided for each EDO in the periodic table, we rely on reaching both upwards from the previous nominal and downwards from the next nominal, providing enough symbols so that these two extensions at least touch in the middle. For most notations, the symbol set terminates at the point where this happens: either touching (in the case of an odd count of steps), or overlapping by one (in the case of an even count of steps). But for many limma fraction based notations (16, 21, 23, 28, 33, 35, 47) more symbols than should be necessary by this process are provided. 

When I tested things in Scala using the minimal sets, however, Scala inserted sharp and flat symbols (in the case of mixed; in the case of pure it did the equivalent thing). So it seems that providing these extra symbols is necessary to instruct Scala to do the right thing.

But I think that your inclusion of these extra symbols in the periodic table was for more reason than configuring Scala (well, you may not have even had Scala in mind specifically). I'm thinking that maybe in the case of these limma fraction notations, for some reasons (pertinent to the underlying meanings of the symbols, their capture zones, etc.) it is considered subpar to use C!) in 16EDO rather than B(|\, that is, that all the symbols should be oriented upwards.

Reply:
No. For 16-EDO there are 3 steps to the limma "EF=3", and -1 (i.e. negative) steps to the apotome. No apotome symbol is used with the limma-fraction notations. Not even with 33, 40 or 47. So there can be no "mixed" notation for limma fraction EDOs, except it could be called Evo notation if you only use single-shaft symbols. You seem to be confirming that that is always possible.

Yes, we rely on reaching both upwards from the previous nominal and downwards from the next nominal, but there is no intention to barely meet if odd, or overlap by only 1 if even. Yes, 52 is cutting it fine, but in most cases there are more symbols than are needed, to allow alternative spellings.

On the periodic table, we only show up to the largest single-shaft symbol. The remaining symbols are either looked up as apotome complements (or simply combined with conventional sharps and flats), or looked up as limma complements, of which there are only 2. ||\ is the limma and ||) is the limma minus |(. See near the end of this post. (Note that this post does not have the final notations) http://forum.sagittal.org/viewtopic.php?p=781#p781

There should be no bias towards using upward symbols over downward. But yes, Scala will need some code added to deal with limma fraction notations.

Reply:
You're right, there are 3 steps to the limma in 16-EDO. What I _meant_ to say was that therefore there are two degrees between an E and an F. And my point was that only one symbol should be required to notate 16-EDO, since you'd use the upwards version of it for one of those two degrees and the downwards version of it for the other. 

I can double-check, but I am fairly sure that only for EDOs 16, 21, 23, 28, 33, 35, 47 was it the case that more symbols were provided than necessary in this way. So I wouldn't call that "most cases". However, "there is no intention to barely meet if odd, or overlap by only 1 if even" is 100% clear. I agree it's nice to support alternative spellings when possible. 

And I agree there should not be any bias toward upward or downward symbols. It was only an extremely-low-confidence guess as to what the intent was, given that so few of the EDO's demonstrated this excess.

So I take it I got it right, in the .par file changes then?

Reply:
But your crazy-beautiful Google sheet clearly shows how 52edo is a worst case re overlap, and you get more and more overlap the further you go left on the periodic table. For example, in 22edo, every note between B and D can be notated as an altered B, C or D.

No the limma fraction notations are not right. You must add N\ to those less than 35, and you must add N) N\ to 35, 40 and 47, because they have |( .

Reply:
See http://forum.sagittal.org/viewtopic.php?p=1747#p1747 for the full limma-fraction notations.

Reply:
Okay, as discussed on the other conversation on this PR re: limma-fraction notations, I will get those N) and N\'s in.

As an aside, was X\ ever proposed as a "whole tone" symbol? As it is /N\ + N\ ? Or was there never any need?

Reply:
I understand markdown ate your backslash and you meant you'll get those N) and N\\'s in.

Reply:
That slosh-eating by markdown is a classic violation of the keep-the-simple-things-simple principle. People who have no interest in the more complex possibilities offered by markdown are hit with the additional cognitive load of having to remember to double their sloshes (or was it their slashes? ;-)).

Reply:
Guhhh. Yes, of course, that's what I meant. 

If I'm feeling charitable, I'd concede that perhaps the target audience of this tool, professional software developers such as myself, may typically be _more_ used to escaping their backslashes than not.